nxcomp/src/Loop.cpp

```diff
@@ -3573,7 +3573,7 @@ int SetupAuthInstance()
 
   if (control -> ProxyMode == proxy_server)
   {
-    if (authCookie != NULL && *authCookie != '\0')
+    if (*authCookie != '\0')
```

Comment:
Alright, it is always true because it is defined a fixed char array. I am astonished that this is the only place where a NULL check is done for those arrays...